examples/pytorch/question-answering/run_qa.py

```diff
@@ -44,6 +125,11 @@
 from transformers.utils import check_min_version
 from utils_qa import postprocess_qa_predictions
 
+# Start SparseML integration
```

Comment:
Since we're not using the custom scripts anymore and are instead putting the code directly in with git, let's remove all the comment sections for start and end of SparseML integration